packages/ilios-common/addon/components/course/collapsed-objectives.hbs

```diff
@@ -12,11 +7,12 @@
       data-test-title
       {{on "click" @expand}}
     >
-      {{t "general.objectives"}} ({{get this.objectives "length"}})
+      {{t "general.objectives"}}
+      ({{get this.objectives "length"}})
```

Comment:
works, but can be streamlined (assuming that the ouptut of `objectives` remains an array/array-like, see comments on the corresponding component class below).
```suggestion
      ({{this.objectives.length}})
```